Agreements/Airport/IoM_NS.txt

```diff
@@ -0,0 +1 @@
+COPX:*:*:CASEL:EGNS:*:Isle of Man 3(DB-285):NSAPP:*:7000:CASEL
```

Comment:
'Isle of Man' will work with this; it has the same sector ownership order as 3 and just keeps it a little tidier

Reply:
Should this be FL100? 

![image](https://user-images.githubusercontent.com/2829581/61743485-4fa94c80-ad8d-11e9-8ba6-c04aef6f8492.png)

Reply:
@cpawley

Reply:
Revised to FL100, in Agreements/Airport/IoM_NS.txt

Available [here](https://github.com/VATSIM-UK/UK-Sector-File/blob/9712ea197b778ac32bb01f01bd16a64afb3e15aa/Agreements/Airport/IoM_NS.txt)